ui/StatusQ/src/StatusQ/Components/StatusCommunityCard.qml

```diff
@@ -338,39 +379,46 @@ Rectangle {
                 }
             }
 
-            StatusRollArea {
-                Layout.alignment: Qt.AlignVCenter
-                Layout.fillWidth: true
+            // Bottom Row extra info component
+            Loader {
+                id: bottomRowLoader
+                Layout.fillWidth: (item.width===0)
+                Layout.preferredHeight: 24
+                active: ((root.categories.count > 0) || !!root.bottomRowComponent)
+                sourceComponent: tagsListComponent
+            }
 
-                visible: root.categories.count > 0
-                arrowsGradientColor: d.cardColor
-
-                // TODO: Replace by `StatusListItemTagRow` - To be done!
-                content: Row {
-                    spacing: 8
-                    clip: true
-
-                    Repeater {
-                        model: root.categories
-                        delegate: StatusListItemTag {
-                            bgColor: "transparent"
-                            bgRadius: 20
-                            bgBorderColor: Theme.palette.baseColor2
-                            height: 24
-                            closeButtonVisible: false
-                            asset.emoji: model.emoji
-                            asset.width: 24
-                            asset.height: 24
-                            asset.color: "transparent"
-                            asset.isLetterIdenticon: true
-                            title: model.name
-                            titleText.font.pixelSize: 13
-                            titleText.color: d.fontColor
+            Component {
+                id: tagsListComponent
+                StatusRollArea {
+                    arrowsGradientColor: d.cardColor
+
+                    // TODO: Replace by `StatusListItemTagRow` - To be done!
```

Comment:
Just do it here :)

Reply:
This is from `534a0d0f45 (Patryk Osmaczko 2022-10-17 09:54:29 +0200) // TODO: Replace by `StatusListItemTagRow` - To be done!` and I can't see to find `StatusListItemTagRow` so I guess it's still a TODO.